buildflow/core/app/runtime/actors/runtime.py

```diff
@@ -87,24 +104,25 @@ def run(self, *, processors: Iterable[ProcessorAPI]):
         # TODO: these can fail sometimes when the converter isn't provided correctly.
         # i.e. a user provides a type that we don't know how to convert for a source /
         # sink. Right now we just log the error but keep trying.
-        for actor in self._processor_pool_actors:
+        for processor_pool in self._processor_pool_refs:
             # Ensure we can start the actor. This might fail if the processor is
             # misconfigured.
-            actor.run.remote()
-            actor.add_replicas.remote(self.options.num_replicas)
+            processor_pool.actor_handle.run.remote()
+            processor_pool.actor_handle.add_replicas.remote(self.options.num_replicas)
 
         self._runtime_loop_future = self._runtime_autoscale_loop()
 
     async def drain(self) -> bool:
         if self._status == RuntimeStatus.DRAINING:
             logging.warning("Received drain single twice. Killing remaining actors.")
-            [ray.kill(actor) for actor in self._processor_pool_actors]
+            [ray.kill(actor) for actor in self._processor_pool_refs]
```

Comment:
I think this will throw an error since this contains a processor pool ref, not an actor

Reply:
done.